agent/agent_endpoint_test.go

```diff
@@ -1654,6 +1654,49 @@ func TestAgent_ForceLeave_ACLDeny(t *testing.T) {
 	})
 }
 
+func TestAgent_ForceLeavePrune(t *testing.T) {
+	t.Parallel()
+	a1 := NewTestAgent(t, t.Name(), "")
+	defer a1.Shutdown()
+	a2 := NewTestAgent(t, t.Name(), "")
```

Comment:
```suggestion
	a2 := NewTestAgent(t, t.Name()+"-a2", "")
```
Diagnostic improvement: this can help tell the logs apart if the test fails.